hugo/package.json

```diff
@@ -6,6 +6,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "atomic-algolia": "0.3.19",
```

Comment:
This appear to be a somewhat random package. Is there a reason not to use an official Algolia client?

Reply:
This package uses the algolia client. As far as I know algolia client does have enpoints you can call but you still have to do the logic whether to update, add, delete somethings yourself. This package loops through the json and checks if an entry needs to be added, updated or deleted from algolia. It's a small package so we chose to use this instead of building the same thing ourselves

Reply:
Why not use the Algolia CLI? 

https://github.com/algolia/setup-algolia-cli

Reply:
@anne-usmedia Maybe you can have a look if it's easy to use that one instead of atomic-algolia?

Reply:
Discussed with @myitcv. Since the CLI makes it hard to delete records, we are going to use Atomic for now. If needed, we can build it ourselves in the future, but for now this will do.